sources/packages/forms/src/form-definitions/educationprogram.json

```diff
@@ -1655,84 +3368,242 @@
+          "showWordCount": false,
+          "allowMultipleMasks": false,
+          "inputType": "radio",
+          "fieldSet": false,
+          "id": "esmuskn",
+          "defaultValue": ""
+        },
+        {
+          "label": "ESL Error",
+          "tag": "p",
+          "className": "alert alert-warning fa fa-exclamation-circle",
+          "attrs": [
+            {
+              "attr": "",
+              "value": ""
+            }
+          ],
+          "content": "<strong> This program is ineligible for StudentAid BC funding.</strong><br/>\nESL can’t exceed 20% of course content.",
+          "refreshOnChange": false,
+          "customClass": "banner-warning",
+          "hidden": false,
+          "modalEdit": false,
+          "key": "html",
+          "tags": [],
+          "properties": {},
+          "conditional": {
+            "show": true,
+            "when": "eslEligibility",
+            "eq": "20OrMore",
+            "json": ""
+          },
+          "customConditional": "",
+          "logic": [],
+          "attributes": {},
+          "overlay": {
+            "style": "",
+            "page": "",
+            "left": "",
+            "top": "",
+            "width": "",
+            "height": ""
+          },
+          "type": "htmlelement",
+          "input": false,
+          "tableView": false,
+          "placeholder": "",
+          "prefix": "",
+          "suffix": "",
+          "multiple": false,
+          "defaultValue": null,
+          "protected": false,
+          "unique": false,
+          "persistent": false,
+          "clearOnHide": true,
+          "refreshOn": "",
+          "redrawOn": "",
+          "dataGridLabel": false,
+          "labelPosition": "top",
+          "description": "",
+          "errorLabel": "",
+          "tooltip": "",
+          "hideLabel": false,
+          "tabindex": "",
+          "disabled": false,
+          "autofocus": false,
+          "dbIndex": false,
+          "customDefaultValue": "",
+          "calculateValue": "",
+          "calculateServer": false,
+          "widget": null,
+          "validateOn": "change",
+          "validate": {
+            "required": false,
+            "custom": "",
+            "customPrivate": false,
+            "strictDateValidation": false,
+            "multiple": false,
+            "unique": false
+          },
+          "allowCalculateOverride": false,
+          "encrypted": false,
+          "showCharCount": false,
+          "showWordCount": false,
+          "allowMultipleMasks": false,
+          "id": "exiqgg3"
+        }
+      ],
+      "id": "elp3zbo"
+    },
+    {
+      "label": "Program partnerships",
+      "tag": "p",
+      "className": "",
+      "attrs": [
+        {
+          "attr": "",
+          "value": ""
+        }
+      ],
+      "content": "Program partnerships",
+      "refreshOnChange": false,
+      "customClass": "category-header-medium primary-color has-sub-heading",

```

Comment:
`has-sub-heading`  can we give a better name if the class

Reply:
has-sub-heading removed from forms and global styles. used mt-n2 bootstrap as discussed.